packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -181,7 +181,6 @@ function InstalledFonts() {
 							) ) }
 						</>
 					) }
-					<Spacer margin={ 16 } />
```

Comment:
I think this was added to create some space at the bottom of the content. I don't think this is needed now that it's possible to see the bottom of the content above the footer.